doc/orm_tutorial.rst

```diff
@@ -361,6 +361,19 @@ measurement functions can be applied to
     >>> print '%s: %f' % (lake.name, bufferarea)
     Majeur: 21.485781
 
+Automatically use a function at insert or select
+------------------------------------------------
+
+Sometimes the application always want to apply a function in an insert or in a select.
```

Comment:
Having both "sometimes" and "always" here sounds strange to.

Also s/want/wants